src/pages/busca-ativa/consulta-criancas-estudantes-ausentes/list/historico/index.tsx

```diff
@@ -75,50 +130,173 @@ const BuscaAtivaHistoricoRegistroAcoes: React.FC = () => {
     setLoading(false);
   }, [anoLetivo, codigoAluno, turmaCodigo, obterFrequenciaGlobalAluno]);
 
+  const salvarDadosResponsavel = useCallback(async () => {
+    formResponsavel.validateFields().then(async () => {
+      const response = await responsavelService.atualizarDadosResponsavel(
+        formResponsavel.getFieldsValue(true),
+      );
+      if (response.sucesso) {
+        setModalOpen(false);
+        obterDados();
+      }
+    });
+  }, [formResponsavel]);
+
   useEffect(() => {
     obterDados();
   }, [obterDados]);
 
+  useEffect(() => {
+    formResponsavel.resetFields();
+  }, [formResponsavel, formInitialValues]);
+
+  const abrirModal = () => {
+    dadosResponsavelParaAtualizar();
+    setModalOpen(true);
+  };
+  const onClickCancelar = async () => {
+    if (formResponsavel.isFieldsTouched()) {
+      const confirmou = await confirmar(
+        'Atenção',
+        'Você não salvou as informações preenchidas.',
+        'Deseja realmente cancelar as alterações?',
+      );
+      if (confirmou) {
+        formResponsavel.resetFields();
+      }
+    } else {
+      setModalOpen(false);
+    }
+  };
   const onClickVoltar = () => navigate(ROUTES.BUSCA_ATIVA_CONSULTA_CRIANCAS_ESTUDANTES_AUSENTES);
   const onClickNovoRegistroAcao = () =>
     navigate(ROUTES.BUSCA_ATIVA_HISTORICO_REGISTRO_ACOES_NOVO, {
       state: registroAcaoBuscaAtivaResposta,
     });
 
   return (
-    <Col>
-      <HeaderPage title="Registro de ações">
-        <Col span={24}>
-          <Row gutter={[8, 8]}>
-            <Col>
-              <BotaoVoltarPadrao onClick={() => onClickVoltar()} />
-            </Col>
-            <Col>
-              <ButtonPrimary
-                id={SGP_BUTTON_BUSCA_ATIVA_NOVO_REGISTRO_ACAO}
-                disabled={!podeIncluir || somenteConsulta}
-                onClick={() => onClickNovoRegistroAcao()}
-              >
-                Novo registro de ação
-              </ButtonPrimary>
+    <>
+      <Modal
+        title="Atualizar dados do responsável"
+        centered
+        open={modalOpen}
+        onOk={() => salvarDadosResponsavel()}
+        onCancel={() => onClickCancelar()}
+        destroyOnClose
+        cancelButtonProps={{ disabled: loading, id: SGP_BUTTON_CANCELAR_MODAL }}
+        okButtonProps={{ disabled: loading, id: SGP_BUTTON_SALVAR_MODAL }}
+        okText="Alterar"
+        cancelText="Cancelar"
+        width={1100}
+      >
+        <Form
+          form={formResponsavel}
+          layout="vertical"
+          autoComplete="off"
+          onFinish={salvarDadosResponsavel}
+          validateMessages={validateMessages}
+          initialValues={formInitialValues}
+        >
+          <Grid cols={12}>
+            <div className="row">
+              <Grid cols={6} className="mb-2">
+                <Form.Item name="nome" label="Responsável">
```

Comment:
remover  Form.Item  e utilizar o https://ant.design/components/typography quando for exibir somente label/texto

Reply:
Adicionado typography